### PR TITLE
Add did_client_lose_job_being_in_custody step

### PR DESCRIPTION
**Pull request by arthurashman**
## Description of change
Add new step behind feature flag
Add to income decision tree, but not in flow yet
Add route with alias, controller, form and errors

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-683

## Screenshots of changes (if applicable)
image

## How to manually test the feature
run `rails db:migrate`
Start a new application 
Go to /steps/income/did_client_lose_job_being_in_custody

### CODE REVIEW

File: config/locales/en/steps.yml [hunk 1]
```diff
@@ -125,6 +125,7 @@ en:
             applicant:
               home_address: Enter your client’s home address
               correspondence_address: Enter your client’s correspondence address
+
```
Comment (arthurashman) [inline config/locales/en/steps.yml:128]: These were intentional, just for consistency

File: config/locales/en/helpers.yml [hunk 2]
```diff
@@ -99,6 +101,8 @@ en:
         expert_examination_justification: *ioj_justification_hint
         interest_of_another_justification: *ioj_justification_hint
         other_justification: *ioj_justification_hint
+      steps_income_lost_job_in_custody_form:
+        date_job_lost: For example, 27 3 2007
       steps_evidence_upload_form:
         #TODO update max file size and overall limit
         upload_files: The maximum file size is XMB. Files must be a JPG, BMP, PNG, TIF or PDF.
```
Comment (hiboabd) [inline config/locales/en/helpers.yml:108]: Not related to your PR but I think this can be deleted

**Review by hiboabd — APPROVED** [review]
🚀

File: app/services/decisions/income_decision_tree.rb [hunk 3]
```diff
@@ -2,10 +2,9 @@ module Decisions
   class IncomeDecisionTree < BaseDecisionTree
     def destination
       case step_name
-      # :nocov:
-      when :x
-        edit(:y)
-      # :nocov:
```
Comment (mseedat-moj) [inline app/services/decisions/income_decision_tree.rb:8]: 👍

**Review by mseedat-moj — APPROVED** [review]
Nice

File: config/locales/en/helpers.yml (hunk 2 above)
Comment (arthurashman) [reply config/locales/en/helpers.yml:108]: I'll do it in my branch I've started for the next page 👌

**Pull request merged**